### PR TITLE
ENH: Added bubblegum, renamed dataportal

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by tacaswell** [comment]
git is very very confused moving around all of these identical files....` {service_identity → master/lmfit}/build.sh`

**Comment by ericdill** [comment]
I think this is good to go (or at least ready for review(

File: recreate_binstar_yml.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    print("Usage: python recreate_binstar_yml.py user_account channel_name")
+    print("Example: python recreate_binstar_yml.py edill main")
+    UPLOAD_ACCOUNT = sys.argv[1]
+    try:
+        CHANNEL = sys.argv[2]
+    except IndexError:
+        # use the default channel
+        CHANNEL = 'dev'
+    # make the yaml files
+    make_ymls(os.getcwd())
```
Comment (tacaswell) [inline recreate_binstar_yml.py]: can we pass `UPLOAD_ACCOUNT` and `CHANNEL` into `make_ymls` as args?  Sure this works, but it feel dirty.....

Comment (ericdill) [inline recreate_binstar_yml.py]: If you insist...

**Pull request merged**